wfe2/wfe.go

```diff
@@ -309,6 +311,7 @@ func (wfe *WebFrontEndImpl) Handler() http.Handler {
 	wfe.HandleFunc(m, buildIDPath, wfe.BuildID, "GET")
 	wfe.HandleFunc(m, rolloverPath, wfe.KeyRollover, "POST")
 	wfe.HandleFunc(m, newOrderPath, wfe.NewOrder, "POST")
+	wfe.HandleFunc(m, orderPath, wfe.Order, "GET")
```

Comment:
Can you add this endpoint to `wfe2/wfe_test.go`'s `TestHTTPMethods` unit test?

Reply:
Done.